zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/management/ClusterEndpointIT.java

```diff
@@ -75,6 +75,34 @@ void shouldRequestPartitionLeave() {
     }
   }
 
+  @Test
+  void shouldRequestClusterPurge() {
+    final ClusterActuator actuator;
+    try (final var cluster = createCluster(2)) {
+      // given
+      cluster.awaitCompleteTopology();
+      actuator = ClusterActuator.of(cluster.availableGateway());
+
+      // when -- request a purge
+      final var response = actuator.purge(false);
+
+      // then
+      assertThat(response.getPlannedChanges().stream().map(Operation::getOperation))
+          .containsExactlyElementsOf(
+              List.of(
+                  OperationEnum.PARTITION_LEAVE,
+                  OperationEnum.PARTITION_LEAVE,
+                  OperationEnum.PARTITION_LEAVE,
+                  OperationEnum.PARTITION_LEAVE,
+                  OperationEnum.DELETE_HISTORY,
+                  OperationEnum.DELETE_HISTORY,
```

Comment:
:question: Why are we deleting twice? Shouldn't we only delete the history once on the coordinator node, deleting for all partitions?